lavamoat/build-system/policy-override.json

```diff
@@ -3,13 +3,14 @@
     "@babel/core": {
       "packages": {
         "<root>": true,
-        "@babel/preset-env": true,
-        "@babel/preset-react": true,
-        "@babel/plugin-transform-runtime": true,
         "@babel/plugin-proposal-class-properties": true,
+        "@babel/plugin-proposal-nullish-coalescing-operator": true,
         "@babel/plugin-proposal-object-rest-spread": true,
         "@babel/plugin-proposal-optional-chaining": true,
-        "@babel/plugin-proposal-nullish-coalescing-operator": true
+        "@babel/plugin-transform-runtime": true,
+        "@babel/preset-env": true,
+        "@babel/preset-react": true,
+        "@babel/preset-typescript": true
```

Comment:
I re-alphabetized this list, but `@babel/preset-typescript` is the new entry here.